plugins/ragdoll/plugin/commands/sh_fall.lua

```diff
@@ -6,7 +6,12 @@ COMMAND.aliases = { 'fallover', 'charfallover' }
 COMMAND.no_console = true
 
 function COMMAND:on_run(player, delay)
-  delay = math.clamp(tonumber(delay) or 0, 2, 60)
+  delay = math.clamp(tonumber(delay) or 0, 0, 60)
+
+  if delay > 0 and delay < 2 then
```

Comment:
Instead of throwing an error, can you just make it so if it's not 0, but less than 2, it clamps to 2?